pixl_core/README.md

```diff
@@ -98,7 +98,7 @@ for convenience `latest` is a symlink to the most recent extract.
 ## Project configuration
 
 The `project_config` module provides the functionality to handle
-[project configurations](../README.md#configure-a-new-project).
+[project configurations](../README.md#configure-a-new-project). <== SK comment I'm not sure this goes to exactly the right place OR the name in the # is misleading
```

Comment:
Also confused by this one.

@stefpiatek – should this be a hyperlink to PIXL/projects/configs?